shared/css/color.scss

```diff
@@ -153,6 +153,7 @@ $brand_accent_default: #ED6060;
 
 //Product
 $product_affirmative_default: #3EA33E;
+$product_affirmative_100: #E2F1E2;
```

Comment:
This color was in the figma but not in this file. It was named "light-affirmative-100" in the figma so I named it similarly. Happy to rename if anyone has suggestions!